src/System.Text.Encoding/tests/UTF32Encoding/UTF32EncodingDecode.cs

```diff
@@ -101,28 +101,28 @@ public void Decode_InvalidBytes()
             Decode_InvalidBytes(new byte[] { 0x00, 0x00, 0x00, 0xFF }, 0, 4, "\uFFFD");
             Decode_InvalidBytes(new byte[] { 0xFF, 0xFF, 0xFF, 0xFF }, 0, 4, "\uFFFD");
         }
-
-        public static byte[] GetBigEndianBytes(byte[] littleEndianBytes)
+        
+        public static byte[] GetBigEndianBytes(byte[] littleEndianBytes, int index, int count)
```

Comment:
why we having index and count parameters at all for GetBigEndianBytes while not used in the method body?